packages/babel-plugin-transform-es3-reserved-words/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "babel-plugin-transform-es3-reserved-words",
+  "version": "7.0.0-beta.2",
```

Comment:
Couple things:

1. Rename to `@babel/plugin-transform-es3-reserved-words`
2. Match version to `7.0.0-beta.3`
3. Set devDep to `@babel/helper-plugin-test-runner` and match its version to `7.0.0-beta.3`